src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -19,6 +19,34 @@ const generateCommunityForumNavItem = (productSlug?: ProductSlug) => {
   return { href, title }
 }
 
+/**
+ * A dictionary of product slugs to GitHub links.
+ */
+const GITHUB_LINKS_BY_SLUG: { [K in ProductSlug]: string } = {
+  boundary: 'https://github.com/hashicorp/boundary',
+  consul: 'https://github.com/hashicorp/consul',
+  hcp: 'https://github.com/hashicorp', // No specific repo to link to
+  nomad: 'https://github.com/hashicorp/nomad',
+  packer: 'https://github.com/hashicorp/packer',
+  sentinel: 'https://github.com/hashicorp', // No specific repo to link to
+  terraform: 'https://github.com/hashicorp/terraform',
+  vagrant: 'https://github.com/hashicorp/vagrant',
+  vault: 'https://github.com/hashicorp/vault',
+  waypoint: 'https://github.com/hashicorp/waypoint',
+}
+
+/**
+ * Generates the sidebar nav item for the Community Forum link. If given a
+ * Product slug specified in `COMMUNITY_FORUM_LINKS_BY_SLUG`, that Product's
+ * link will be used. Otherwise, a general default link is used.
+ */
+function generateGitHubNavItem(productSlug?: ProductSlug) {
+  return {
+    title: 'GitHub',
+    href: GITHUB_LINKS_BY_SLUG[productSlug] || 'https://github.com/hashicorp',
```

Comment:
💭 Thoughts on putting this default in a constant like `DEFAULT_COMMUNITY_FORUM_LINK`?